pkgs/development/libraries/kirigami-addons/default.nix

```diff
@@ -12,14 +12,14 @@
 
 mkDerivation rec {
   pname = "kirigami-addons";
-  version = "0.6";
+  version = "0.7.2";
```

Comment:
Apps using kirigami-addons should also be ran, as in my experience it's possible that non-backward compatible changes be introduced that only show up at runtime.

Reply:
I've tested neochat and tokodon and they seem to work fine.

Reply:
(both 23.01 and 23.04)

Reply:
Don't take this as a task to do by yourself. I've just listed all apps depending on it. I'll try to test this PR soon~ish, and see if it is all good for those.

 - [x] neochat
 - [x] tokodon         
 - [ ] alligator
 - [ ] angelfish
 - [ ] kalendar
 - [ ] kasts
 - [ ] kclock
 - [ ] krecorder
 - [ ] ktrip
 - [ ] kweather
 - [ ] plasma-dialer
 - [ ] plasma-phonebook
 - [ ] plasma-settings
 - [ ] qmlkonsole
 - [ ] spacebar